api/v1alpha1/openapi.yaml

```diff
@@ -5082,6 +5082,8 @@ components:
             - ResourceDeletionFailed
             - DeviceDecommissioned
             - DeviceDecommissionFailed
+            - DeviceApproved
```

Comment:
Should these be `EnrollmentRequestApproved` and `EnrollmentRequestApprovalFailed`?
For the same reason of Avishay's comment.